examples/trials/network-morphism/README.md

```diff
@@ -0,0 +1,10 @@
+# Network Morphism for Automatic Model Architecture Search 
+This example shows us how to use Network Morphism to find good model architectures for deep learning.
```

Comment:
use Network Morphism  tuner?

Reply:
I mean using NetworkMorphismTuner to search for good architecture.